csuntechlab/iSTART
# STR-39 -- FE: Add title tags to front-end routes

# Description
Created and implemented a mix-in for changing route titles.

# Testing
1. Visit Home to ensure the title is 'iSTART'.
2. Visit About to ensure the title is 'iSTART - About'.
